tfx_bsl/tfxio/parquet_tfxio.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TFXIO implementation for Parquet."""
+
+import copy
+from typing import Optional, List, Text, Any
+
+import apache_beam as beam
+import pyarrow as pa
+import pyarrow.parquet as pq
+import tensorflow as tf
+from apache_beam.io.filesystems import FileSystems
+from tensorflow_metadata.proto.v0 import schema_pb2
+
+from tfx_bsl.coders import csv_decoder
+
+from tfx_bsl.tfxio import dataset_options, tensor_adapter, tensor_representation_util, telemetry
+from tfx_bsl.tfxio.tfxio import TFXIO
+
+_PARQUET_FORMAT = "parquet"
+
+
+class ParquetTFXIO(TFXIO):
+  """TFXIO implementation for Parquet."""
+
+  def __init__(self,
```

Comment:
actually, one more small usability thing: can we make all args after file_pattern and column_names key-word only, i.e. `def __init__(self, file_pattern:..., column_names:..., *, min_bundle_size:...,...`

Reply:
done it https://github.com/tensorflow/tfx-bsl/pull/52/commits/ae1b00fe22a566f8e4d05276d6af798c381bc294.

Reply:
Thanks! yeah, that's unfortunate, but I'd like to have it moving forward: I think it helps with preventing some errors